openml/runs/run.py

```diff
@@ -57,7 +57,9 @@ class OpenMLRun(OpenMLBase):
     run_id: int
     description_text: str, optional
         Description text to add to the predictions file.
-        If left None,
+        If left None, is set to the time the arff file is generated.
+    run_details: str, optional (default=None)
```

Comment:
This needs to also be used in the constructor in `openml.functions.runs._create_run_from_xml`.

Reply:
I forgot about this, but it's because [it's not in the xml](https://github.com/openml/OpenML/issues/1087).
I'll just retrieve it safely (with a default value) so we can do a release if there's no response tomorrow (and then should work when/if the server gets an update). Ok?